.github/workflows/codeql-analysis.yml

```diff
@@ -6,17 +6,31 @@ on:
   pull_request:
     branches: [ master ]
   schedule:
+    # Check every Saturday at 18:36
+    #         ┌───────────── minute (0 - 59)
+    #         │ ┌───────────── hour (0 - 23)
+    #         │ │ ┌───────────── day of the month (1 - 31)
+    #         │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #         │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #         │ │ │ │ │
+    #         │ │ │ │ │
+    #         │ │ │ │ │
+    #         * * * * *
     - cron: '36 18 * * 6'
 
 jobs:
   analyze:
-    name: ${{ matrix.c }} & ${{ matrix.cpp }} - Analyze
+    name: ${{ matrix.c }} & ${{ matrix.cpp }} - ${{ matrix.language }}
     runs-on: ubuntu-20.04
 
     strategy:
       fail-fast: false
       matrix:
+        # c is included in cpp.
         language: [ 'cpp', 'python' ]
+        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python' ]
+        # Learn more:
+        # https://docs.github.com/en/free-pro-team@latest/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning#changing-the-languages-that-are-analyzed
```

Comment:
This is not relevant. `# c is included in cpp.` strikes the right balance between too little information and unnecessary information. You could add that there is no separate C language on CodeQL. That would clearly document of what what you considered when you wrote `[ 'cpp', 'python' ]`.